src/System.Drawing.Common/tests/Imaging/ImageFormatTests.cs

```diff
@@ -82,6 +94,15 @@ public void ToString_ReturnsExpected(string expected, ImageFormat imageFormat)
             Assert.Equal(expected, imageFormat.ToString());
         }
 
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Bug fix in .NET Core that is not in NETFX yet, dotnet/corefx 16463")]
+        [ConditionalTheory(Helpers.GdiplusIsAvailable)]
+        [MemberData(nameof(ImageFromFileToStringTestData))]
+        public void Image_RawFormat_ToString(string path, string expected)
+        {
+            var img = Image.FromFile(path);
+            Assert.Same(img.RawFormat.ToString(), expected);
```

Comment:
Nit: swap the arguments... Assert.Same's first argument is expected, second is actual